README.rst

```diff
@@ -0,0 +1,34 @@
+What is Kolibri?
+----------------
+
+Kolibri is an adaptable measured learning environment. A user can install Kolibri and serve the app on a local network,
```

Comment:
I would suggest "Kolibri is a Learning Management System / Learning App designed to run on low-power devices, targeting the needs of learners and teachers in contexts with limited infrastructure."